manager/src/main/resources/application.properties

```diff
@@ -11,7 +11,7 @@ quarkus.swagger-ui.always-include=true
 quarkus.smallrye-openapi.store-schema-directory=target/generated/
 quarkus.smallrye-openapi.info-title=Red Hat Openshift SmartEvents Fleet Manager
 quarkus.smallrye-openapi.info-version=0.0.1
-quarkus.smallrye-openapi.info-description=The api exposed by the fleet manager of the SmartEvents service.
+quarkus.smallrye-openapi.info-description=The API exposed by the fleet manager of the SmartEvents service.
```

Comment:
Please update also the saved file to match 
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/openapi/openapi.yaml#L5